Header header
trajectory_msgs/JointTrajectoryPoint point 
bool finished
